### PR TITLE
Update cypress test

### PR DESCRIPTION
**Pull request by k-pammi**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{
-    "baseUrl": "https://giftaid-staging.comicrelief.com"
+{   //this is the temp envt, should change this to localhost later
+    "baseUrl": "https://5ba4dbef1f12b704899145bd--giftaid-react.netlify.com/update"
```
Comment (AndyEPhipps) [inline cypress.json]: Hmmm, I think `http://localhost:3000` will work everywhere, regardless of domain name

File: cypress/integration/app_spec.js [hunk 2]
```diff
@@ -6,85 +6,224 @@
...
+        cy.get('#footer-menu > li:nth-child(4) > a').should('contain', 'Legal')
+        cy.get('#footer-menu > li:nth-child(5) > a').should('contain', 'Press Area')
+        cy.get('#footer-menu > li:nth-child(6) > a').should('contain', 'Reporting concerns')
+        cy.get('#footer-menu > li:nth-child(7) > a').should('contain', 'Update your preferences')
+        cy.get('#footer-menu > li:nth-child(8) > a').should('contain', 'Privacy')
+        cy.get('#footer-menu > li:nth-child(9) > a').should('contain', 'Cookies')
+        cy.get('#footer-menu > li:nth-child(10) > a').should('contain', 'FAQs')
+        cy.get('#footer-menu > li:nth-child(11) > a').should('contain', 'Accessibility')
+        cy.get('#footer-menu > li:nth-child(12) > a').should('contain', 'Sitemap')
+        cy.get('#footer-menu > li:nth-child(13) > a').should('contain', 'Modern slavery and human trafficking')
```
Comment (AndyEPhipps) [inline cypress/integration/app_spec.js]: I guess it's as valid as any other test, but seeing as **this** content is just being pulled from the live CR.com footer, this very specific test will just fail as soon as that is updated.

I wonder if _not_ checking the order (the nth-child selectors) or checking for just a few of these items would give the same value but more flexibility?

Comment (k-pammi) [reply cypress/integration/app_spec.js]: yeah corecct, will change that to be specific

**Comment by k-pammi** [comment]
@pvhee could you have a look at this please https://circleci.com/gh/comicrelief/giftaid-react/328, it's complaining that XVFB dependency is missing, not sure if there's anything missing in the config above.

File: .circleci/config.yml [hunk 3]
```diff
@@ -7,6 +7,11 @@ jobs:
   build:
     docker:
       - image: node:8
+      # See https://github.com/cypress-io/cypress-docker-images
+      - image: cypress/base:8
```
Comment (pvhee) [inline .circleci/config.yml]: @k-pammi you're using two images, probably you can get rid of `node:8` - to see working cypress tests on Circle check out config in https://github.com/comicrelief/grants-eligibility-form/blob/master/.circleci/config.yml

**Pull request merged**